client/app/layouts/focused-task.scss

```diff
@@ -1,24 +1,54 @@
+@import './../styles/mixins/importer.scss';
 .focused-task {
     position: absolute;
     top: 50%;
     left: 50%;
     transform: translate(-50%, -50%);
-
     padding: 20px;
-
     width: 500px;
     max-width: 500px;
-
     &.focused-task__wide {
         width: 700px;
         max-width: 700px;
     }
 }
 
 .focused-task--header {
```

Comment:
You could nest it in `.focused-task` with `&--header` makes it easier to maintain. Just FYI, no need to update it if you don't want.